cypress/e2e/space-panel/space-panel.spec.ts

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
```

Comment:
This should be your copyright, not ours :)

Reply:
I've copied this from the other test files because all I've checked had this license above them. This is my first time adding a copyright in a PR, should I add my name in the place of "The Matrix.org Foundation C.I.C."?

Reply:
Yup, with your email address if you prefer:

```
Copyright 2022 Your Name
```
or
```
Copyright 2022 Your Name <email@example.org>
```